### ISSUE TITLE
The content is blocked on enabling the AdGuard

### ISSUE THREAD

**Issue by s1tantray** [AdguardTeam/ContentBlocker#402]
Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [yes ] I am running the latest version
- [ yes] I checked the documentation and found no answer

### Issue Details
Precondition: 
Enable Adguard Content Blocker.
Steps:
1.Open Samsung Internet.
2. Visit https://lacertredates.top
3. Check, content is blocked, white screen appears.

- **AdGuard version:**
  v2.8.0
- **Browser:**
  Samsung Internet

### Expected Behavior

Content should be visible.

### Actual Behavior
The content is blocked on enabling the Adguard Content blocker

**Comment by Versty** [comment]
@s1tantray Duplicate of #405
